ngraph/test/util/engine/ie_engines.cpp

```diff
@@ -123,6 +123,41 @@ namespace
     }
 };
 
+namespace
+{
+    InferenceEngine::Precision ng_type_to_precission(const element::Type& target_type)
+    {
+#if defined(__GNUC__) && !(__GNUC__ == 4 && __GNUC_MINOR__ == 8)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic error "-Wswitch"
+#pragma GCC diagnostic error "-Wswitch-enum"
+#endif
+        switch (target_type)
```

Comment:
It's surprising that there was no such switch before.

Reply:
That's right: https://github.com/openvinotoolkit/openvino/blob/ab66eab6528b2ec99cc992f07ffd3dcb2950b8bd/inference-engine/src/plugin_api/ie_ngraph_utils.hpp#L93, but it is not part of public IE API